libpdbg/dtb.c

```diff
@@ -258,8 +299,20 @@ static void bmc_target(struct pdbg_dtb *dtb)
 				dtb->system.fdt = &_binary_p9_dtb_o_start;
 		} else if (!strcmp(pdbg_backend_option, "p10")) {
 			pdbg_proc = PDBG_PROC_P10;
-			if (!dtb->backend.fdt)
-				dtb->backend.fdt = &_binary_bmc_kernel_dtb_o_start;
+			if (!dtb->backend.fdt) {
+				char* system_type = get_p10_system_type();
+				if (strcmp(system_type, "Everest") == 0) {
```

Comment:
Instead of using direct hard coded strings like `Everest` and `Rainier 2U` etc can we define them statically either in file scope or function scope (if file scope is not necessary at all)? Advantage would be that in case we need to change those in future we need to change in only one place.